extension.toml

```diff
@@ -2,7 +2,7 @@ id = "cairo"
 name = "Cairo"
 version = "0.0.2"
 schema_version = 1
-authors = ["Thomas Robert Butler IV trbiv@proton.me"]
+authors = ["Thomas Robert Butler IV trbiv@proton.me", "Okhai Omotuebe okhaio@protonmail.com"]
 description = "Cairo language support for the Zed editor"
 repository = "https://github.com/trbutler4/zed-cairo"
 
```

Comment:
Instead of just updating the for expression, we should update the entire grammar to the latest commit

Reply:
Alright, I'll do this tonight

Reply:
@trbutler4 done!